### PR TITLE
chore(docs): update windows installation docs

### PR DESCRIPTION
**Pull request by tw15egan**
Few updates while running through getting set up on my Windows 10 machine

### CODE REVIEW

**Review by dakahn — APPROVED** [review]
Whoa! I'm super happy this still mostly worked :D

File: docs/guides/setup/windows.md [hunk 1]
```diff
@@ -21,8 +21,8 @@ but can be unexpected.
 6. Install nvm via their install and update script
    [found here](https://github.com/nvm-sh/nvm#install--update-script)
 7. Source your newly NVMed .bashrc `source ~/.bashrc`
-8. Install Node 10 `nvm install 10`
-9. Install Python 2 `sudo apt install python-minimal`
+8. Install Node 12 `nvm install 12`
```
Comment (emyarod) [inline docs/guides/setup/windows.md]:
```suggestion
8. Install latest LTS Node release `nvm install --lts`
```

does this help us avoid bumping the version number in the docs every time it updates?

Comment (tw15egan) [reply docs/guides/setup/windows.md]: Good idea!

**Comment by tw15egan** [comment]
@emyarod does the rest look good?

**Comment by emyarod** [comment]
yep looks good to me!

**Review by emyarod — APPROVED** [review]

**Pull request merged**